index.js

```diff
@@ -124,10 +125,78 @@ function parseFileExtension(opts) {
   return opts;
 }
 
+// mongodb-linux-x86_64-rhel62-3.6.2.tgz
+//
+// Red Hat Enterprise Linux Server release 7.2 (Maipo)
+function parseRHEL(data) {
+  if (data.toString().match('7.1')) return 'rhel71';
+  if (data.toString().match('7.2')) return 'rhel72';
+  if (data.toString().match('6.0')) return 'rhel60';
+}
```

Comment:
there is no fall through case here, what if none of your matches work? It just returned `undefined`, I think you should be more explicit.

Reply:
Good point. I am going to throw an error because it doesn't seem like we should guess and send a version that will possibly not work.